scripts/ci/pre_commit/pre_commit_check_setup_extra_packages_ref.py

```diff
@@ -24,18 +24,18 @@
 import os
 import re
 import sys
-from os.path import dirname
+from pathlib import Path
 
 from rich import print
 from rich.console import Console
 from rich.table import Table
 
-AIRFLOW_SOURCES_DIR = os.path.join(dirname(__file__), os.pardir, os.pardir, os.pardir)
+AIRFLOW_SOURCES_DIR = Path(__file__).resolve().parents[3]
```

Comment:
```suggestion
AIRFLOW_SOURCES_DIR = Path(__file__).parents[3].resolve()
```

or

```suggestion
AIRFLOW_SOURCES_DIR = Path(__file__).parents[3]
```

since the original code seems to work without resolving anyway?

Reply:
There are 13 occurrences of `resolve().parents` in the repo. Should we fix them all?

Reply:
That’d be awesome, although we can focus on the lines changed in this PR first.